zerver/lib/bugdown/__init__.py

```diff
@@ -1160,7 +1160,7 @@ def do_convert(md, realm_domain=None, message=None):
         cleaned = _sanitize_for_log(md)
 
         # Output error to log as well as sending a zulip and email
-        logging.getLogger('').error('Exception in Markdown parser: %sInput (sanitized) was: %s'
+        logging.error('Exception in Markdown parser: %sInput (sanitized) was: %s'
```

Comment:
Ok, I think the easiest thing to do here is to extract something like `log_markdown_error` that is easier to mock.  I can unmock it in my local environment to make sure it still spits out to the screen during tests, then re-mock it to stop being spammy.